packages/app/src/cli/api/graphql/app_active_version.ts

```diff
@@ -39,7 +39,7 @@ interface AppModuleVersion {
   registrationUuid: string
   registrationTitle: string
   type: string
-  specification?: AppModuleVersionSpecification
+  specification: AppModuleVersionSpecification
```

Comment:
@billfienberg no, the `app` module does not export any public API it's only for internal use. If you need to add a breaking change to one exporter method, you only need to make sure that you fix any potential build error inside the `app` module itself

Reply:
Be aware that the graphql schema exports this field as optional [here](https://github.com/Shopify/partners/blob/master/db/graphql/cli_schema.graphql#L293). I'm not sure it could actually be `undefined`, but maybe we should respect the schema. Or if you have more context on this and think this is safe, then go ahead

Reply:
CC @alvaro-shopify